load-fhir-data.js

```diff
@@ -161,7 +161,10 @@ GC.get_data = function() {
             process(vitalsByCode['8302-2'],  units.cm,  p.vitals.lengthData);
             process(vitalsByCode['8287-5'],  units.cm,  p.vitals.headCData);
             process(vitalsByCode['39156-5'], units.any, p.vitals.BMIData);
-            processBoneAge(vitalsByCode['37362-1'], p.boneAge, units);
```

Comment:
I'd add a comment with the difference between these two LOINCs and why we prefer one over the other. Don't mention Cerner.

Reply:
Added the comment.